tests/testthat/examples/testpkg.altdoc/R/examplesIf_false.R

```diff
@@ -0,0 +1,13 @@
+#' Examples If FALSE
+#'
+#' @param x A parameter
+#'
+#' @return Some value
+#' @export
+#'
+#' @examplesIf 2 + 2 == 5
+#' @examples
```

Comment:
You can remove the `@examples` tag

Reply:
https://github.com/etiennebacher/altdoc/pull/271/commits/86fc1edef7f09f3106c77b0cfac6eced41b48818